controllers/etcdcopybackupstask/reconciler.go

```diff
@@ -17,16 +17,19 @@ package etcdcopybackupstask
 import (
```

Comment:
Please write unit tests for the new functions/methods added in the PR, to ensure we avoid regressions in any later refactorings.